yarpcconfig/chooser_test.go

```diff
@@ -1060,6 +1062,70 @@ func TestChooserConfigurator(t *testing.T) {
 	}
 }
 
+func TestChooserConfiguratorWithoutPeerListUpdater(t *testing.T) {
+	tests := []struct {
+		desc    string
+		given   string
+		env     map[string]string
+		wantErr []string
+		test    func(*testing.T, yarpc.Config)
+	}{
+		{
+			desc: "invalid peer list updater",
+			given: whitespace.Expand(`
+				outbounds:
+					their-service:
+						unary:
+							fake-transport:
+								fake-list:
+									bogus-updater: 10
+			`),
+			wantErr: []string{
+				`failed to configure unary outbound for "their-service": `,
+				`no recognized peer list updater in config`,
+				`got bogus-updater`,
+				`no peer list updaters are registered`,
+			},
+		},
+	}
+
+	for _, tt := range tests {
```

Comment:
This test is very similar to `TestChooserConfigurator` but uses a different `configer`. Instead of duplicating all the logic, can we instead allow the test table to specify an "override" `configer` as part of the test-case, so that this can be merged into the above test?

Reply:
done